src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure [] figures = new Figure [6];

```

Comment:
1. no spaces between type and []
2. magic number
```suggestion
        Figure[] figures = new Figure[FIGURES_AMOUNT];
```